portfolio.html

```diff
@@ -43,65 +43,57 @@
+    <div class="w-100 bg-white h3 tc white fixed shadow-5 z-1"></div>
+    <div class="w-100 bg-white h3 mt2 pb3 absolute z-2"></div>
+    <nav class="w-100 bg-transparent-l bg-white tc dwyl-dark-gray relative fixed z-3">
       <input type="checkbox" id="burger" class="absolute top-2 right-1 dn">
       <label for="burger" class="dn-l pointer absolute top-1 right-1">
-        <span class="sr-only">Show menu</span> <i class="fa fa-bars white dib f2"></i>
+        <i class="fa fa-bars dwyl-dark-gray dib f2"></i>
       </label>
-      <ul class="overflow-hidden menu db-l w-100 list tc pl0 pt3 mv0 f3 fw3 f5-l shadow-3">
+      <ul class="overflow-hidden menu db-l w-100 list tc pl0 pt3 mv0 f3 fw3 f5-l">
         <li class="absolute top-1 static-l ph4 mh2 fw3 di-l pt1 pb3 pv3-l">
-          <a href="/" class="white link">
+          <a href="/" class="dwyl-dark-gray link">
             <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl heart logo" class="w1-plus"/>
             dwyl
           </a>
         </li>
-        <li class="ph4-l ph5-xl ph5 pt5 pb2 tl dn-l"><a href="/" class="white link topnav">Home</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/portfolio" class="white link topnav">Portfolio</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/values" class="white link topnav">Values</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/team" class="white link topnav">Team</a></li>
-        <li class="ph4-l ph5-xl ph5 dib-l pv2 tl pv0-l"><a href="/time-app" class="white link topnav">App
-          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pv2 tl pv0-l"><a href="/blog" class="dwyl-mint link topnav">Blog</a></li>
-        <li class="ph4-l ph5-xl ph5 di-l pt2 pb3 tl pv0-l"><a href="#contact" id="contact-link" class="white link topnav">Contact</a></li>
+        <li class="ph4 pt5 pb2 tl dn-l"><a href="/" class="dwyl-dark-gray link">Home</a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/portfolio" class="dwyl-dark-gray link">Portfolio</a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/values" class="dwyl-dark-gray link">Values</a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/team" class="dwyl-dark-gray link">Team</a></li>
+        <li class="ph4 dib-l pv2 tl pv0-l"><a href="/time-app" class="dwyl-dark-gray link">App
+          <span class="dwyl-bg-yellow fw3 pa1 br2 f7 shadow-6">BETA</span></a></li>
+        <li class="ph4 di-l pv2 tl pv0-l"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph4 di-l pt2 pb3 tl pv0-l"><a href="#contact" id="contact-link" class="dwyl-dark-gray link">Contact</a></li>
       </ul>
     </nav>
 
     <section class="bg-white pt6 pb4">
-      <section class="mw9 center ph3">
-        <h1 class="pl3 pr3 f2 w-100 tc tl-ns dwyl-dark-gray">Portfolio</h1>
+      <section class="mw8 center pt5 ph3">
```

Comment:
The padding here looks suddenly massive:
<img width="1109" alt="screen shot 2017-10-09 at 19 31 20" src="https://user-images.githubusercontent.com/4185328/31352966-94c961a4-ad28-11e7-9883-61824a54259f.png">